.github/workflows/publish-release-image-wrapper.yaml

```diff
@@ -0,0 +1,12 @@
+name: "Create and Publish Release Docker Image"
```

Comment:
Would it be a good idea to sync the name of this workflow with the same file you have in your other PR?
This will keep clutter on the actions page to a minimum.

Otherwise LGTM